houston/controller/hook/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * houston/controller/hook/index.js
+ * Handles all outside service inputs
+ *
+ * @exports {Object} - Koa router
+ */
+
+import Router from 'koa-router'
+
+import github from './github'
+import jenkins from './jenkins'
+
+const route = new Router({
+  prefix: '/hook'
+})
+
+route.use(github.routes(), github.allowedMethods())
+route.use(jenkins.routes(), jenkins.allowedMethods())
+
+// Use event hook listeners as well
+require('./atc')
```

Comment:
Why is this here, it's not used in this file. Is this to set up the routing for the websockets?